app/Policies/SignaturePolicy.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Policies;
+
+use App\Models\Signature;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class SignaturePolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     */
+    public function viewAny(User $user): bool
+    {
+        return true;
```

Comment:
Do we really want these visible to everyone with Nova access?

Reply:
🤷🏻‍♂️ I don't think they're particularly sensitive. Do you want to restrict to PM+?

Reply:
`read-users` includes everyone with Nova access

https://my.robojackets.org/nova/resources/permissions/2
https://my.robojackets.org/nova/resources/permissions/83

Reply:
It's fine as-is then. Never mind.